ee/tabby-ui/components/chat/chat-panel.tsx

```diff
@@ -110,10 +111,14 @@ function ChatPanelRenderer(
                 <Badge
                   variant="outline"
                   key={`${activeSelection.filepath}_active_selection`}
-                  className="inline-flex items-center gap-0.5 rounded text-sm font-semibold"
+                  className="inline-flex flex-nowrap items-center gap-1.5 overflow-hidden rounded text-sm font-semibold"
                 >
-                  <span className="text-foreground">
-                    {getContextLabel(activeSelection)}
+                  <ContextLabel
+                    context={activeSelection}
+                    className="flex-1 truncate"
+                  />
+                  <span className="shrink-0 text-muted-foreground">
+                    Current file
```

Comment:
could you try using this icon to see if gives better visual?

https://lucide.dev/icons/text-select